client/src/About.css

```diff
@@ -0,0 +1,31 @@
+.about-card-wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    padding: 40px 0 60px 0;
+    gap: 10px;
+}
+  
+@media only screen and (min-width: 375px) {
```

Comment:
Great, mobile first 👏 
Although 375px as a breakpoint will include almost every device out there, it's more common to introduce breakpoints at higher width to adapt the layout. 
You don't need to repeat all the properties from the default selector above, just the ones that have changed (in this case, looks like padding and margin only)

Reply:
Have made changes so that only the necessary is changed in the media query. Thanks for pointing that out. About the media query and the min-width, ideally, I intend to add another breakpoint at 768px and probably max-width 1400px in future so this was a start tocapture a mobile version for the least. But, I think you are saying that I can start higher for instance at min-width 540px. Is that figure more common, or what would you recommend please?

Reply:
Advice on media queries can differ; in some case breakpoints can be added when the layout requires it, ie when it starts to break or not work so well.
An alternative is to set breakpoints at 'classic' device widths - a very common one is 768px (iPad portrait width). Some advice here - https://css-tricks.com/snippets/css/media-queries-for-standard-devices/